### ISSUE TITLE
CLI Binary Path Bug

### ISSUE THREAD

**Issue by Caiuse** [google-ar/arcore-unity-sdk#237]
Seems that NET4.0 expects a process binary path to be absolute if it is not in System32 directory. I've written a fix which finds the current project directory and appends to the begins of the binary path.

```
string projectPath = Application.dataPath.Substring(0, Application.dataPath.Length - 6);
path = @"" + projectPath + AssetDatabase.GUIDToAssetPath(cliBinaryGuid[0]);
```

**Comment by pablisho** [comment]
Hi, thanks for the report and proposing a solution. I added it to our internal tracker.

**Comment by pablisho** [comment]
Hi, this issue has been fixed in [ARCore Unity SDK v1.3.0](https://github.com/google-ar/arcore-unity-sdk/releases/tag/v1.3.0). Feel free to reopen if you still experience the issue.
